app/assets/stylesheets/pages/_playbook.scss

```diff
@@ -0,0 +1,80 @@
+.pb-bg-navy { background-color: #163963;}
+.pb-footer { background-color: #cedbec; }
+
+.pb-lead {
```

Comment:
i'm not a fan of these custom & page specific font sizes and line-heights -- it feels like 1.5 line height should be sufficient here, which we already have; re: 20px font-size, if we need that, let's add to the typography component